scripts/declarations/lint/index.js

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
```

Comment:
Bugs with deleted files when using -m option. 

Maybe `getModifiedServiceDocumentTypes` should not return deleted services?

Reply:
Agreed